config/locales/en/course/user_invitations.yml

```diff
@@ -3,7 +3,7 @@ en:
     user_invitations:
       errors:
         duplicate_user: '%{user} appears more than once in the submission.'
-        invalid_email: '%{email} is an invalid email address.'
+        invalid_email: "%{email} is invalid or already taken."
```

Comment:
@fonglh Have made it to show the specific message in the second commit. now the second part of the message is generated by rails, in the format of `xxx@emai.com is invliad: Email has already been take`. Any suggestions?